packages/fxa-auth-server/docs/swagger/util-docs.ts

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import TAGS from './swagger-tags';
+
+const GET_RANDOM_BYTES_POST = {
+  notes: [
+    'Get 32 bytes of random data. This should be combined with locally-sourced entropy when creating salts, etc.',
+  ],
+  tags: TAGS.UTIL,
+};
+
+const VERIFY_EMAIL_GET = {
+  tags: TAGS.UTIL,
+};
+
+const COMPLETE_RESET_PASSWORD_GET = {
+  tags: TAGS.UTIL,
+};
```

Comment:
I won't call out others but for example I'd also just stick this in a constant called `TAGS_UTIL`.